### PR TITLE
Forkedpath

### PR DESCRIPTION
**Pull request by lessreal**
Thanks in advance, @casey-collab

### CODE REVIEW

File: english/impossible-adventure/bleep.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bleep bloop bleep... bleep bloop bleep... bleep bloop bleep...
```
Comment (casey-collab) [inline english/impossible-adventure/bleep.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/impossible-adventure/bloop.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+bloop bleep bloop... bloop bleep bloop... bloop bleep bloop...
```
Comment (casey-collab) [inline english/impossible-adventure/bloop.md]: This file has also been added to a directory that already contained a page.

File: english/impossible-adventure/bwetap.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[BWEEEEEEEEEEEEEUW](tapbwe.md)
```
Comment (casey-collab) [inline english/impossible-adventure/bwetap.md]: This file has also been added to a directory that already contained a page.

File: english/impossible-adventure/destroy.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+DESTROY HIM, MY ROBOTS!
```
Comment (casey-collab) [inline english/impossible-adventure/destroy.md]: This file has also been added to a directory that already contained a page.

File: english/impossible-adventure/start.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+[STAY FOREVER!](https://youtu.be/i1_fDwX1VVY)
```
Comment (casey-collab) [inline english/impossible-adventure/start.md]: This file has also been added to a directory that already contained a page.

File: english/impossible-adventure/tapbwe.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+[...tap tap tap tap tap tap...](bloop.md)
```
Comment (casey-collab) [inline english/impossible-adventure/tapbwe.md]: This file has also been added to a directory that already contained a page.

File: english/impossible-adventure/tapbwetap.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+[...tap tap tap tap tap tap...](bloop.md)
```
Comment (casey-collab) [inline english/impossible-adventure/tapbwetap.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by lessreal** [comment]
Hi, @casey-collab - how about this?

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**